context/app/static/js/components/Routes/useSetUrlPathAndSearchBeforeLogin.js

```diff
@@ -0,0 +1,13 @@
+import { useEffect } from 'react';
+import Cookies from 'universal-cookie';
+
+function useSetPathAndSearchBeforeLogin(path) {
+  useEffect(() => {
+    const cookies = new Cookies();
+    if (path !== '/login' && path !== '/logout') {
```

Comment:
On the javascript side, will the path ever be /login or /logout? I had thought those were handled by python.